### PR TITLE
Prepare 1.1 release

### PR DESCRIPTION
**Pull request by hoechenberger**
cc @larsoner

### CODE REVIEW

**Comment by hoechenberger** [comment]
Nice, all import tests are passing!

Only thing I noticed is a DeprecationWarning:

```
/Applications/MNE-Python/.mne-python/lib/python3.10/site-packages/mne_kit_gui/_utils.py:29: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
[659](https://github.com/mne-tools/mne-installers/runs/6736807408?check_suite_focus=true#step:22:660)
  if LooseVersion(version) >= LooseVersion('5.10'):
[660](https://github.com/mne-tools/mne-installers/runs/6736807408?check_suite_focus=true#step:22:661)
```

**Comment by hoechenberger** [comment]
Oh wait, the tests used 1.0.3… hum

**Comment by hoechenberger** [comment]
Aaaand we're green, yay!

**Comment by hoechenberger** [comment]
Aaaand we're green, yay!

**Review by larsoner — CHANGES_REQUESTED** [review]
Awesome, just marking with request changes so we don't forget to remove the dev lines before merge

**Comment by hoechenberger** [comment]
Wondering if we should cut a dev prerelease so we can actually test it locally, WDYT?

**Comment by larsoner** [comment]
That's a good idea, then we can debug any obvious issues before release rather than after

**Comment by hoechenberger** [comment]
Linux errors are unrelated to my changes here, not sure what's going on …

**Comment by larsoner** [comment]
Probably some new missing library. On CircleCI we diagnose/check with this:

https://github.com/mne-tools/mne-python/blob/a9b139d00d6ce966dfebc078597ad335ebb425f9/.circleci/config.yml#L148-L150

A step to do this before the tests/`sys_info` might help

**Comment by hoechenberger** [comment]
@larsoner I wonder if it could be related to this:
https://github.com/conda-forge/staged-recipes/pull/19156

**Comment by hoechenberger** [comment]
Also see https://github.com/conda-forge/qt-main-feedstock/issues/32

**Comment by larsoner** [comment]
Yes it is almost always some `libxcb*` that needs to be installed, see our list for CircleCI in https://github.com/mne-tools/mne-python/blob/a9b139d00d6ce966dfebc078597ad335ebb425f9/tools/setup_xvfb.sh#L14

You could wait for the upstream fix, but the easier/faster thing is just to `sudo apt install` these here. It would be good to add the test line I pointed to above, too, since these issues pop up from time to time, and the `LD_DEBUG` makes it clear what needs to be `apt install`ed to get things working again

**Comment by hoechenberger** [comment]
@larsoner Cann we safely assume that our users will have the required `xcb` libraries installed on their system?

Because if not, I'd rather pin Qt to <5.15 on Linux for now, until the issue has been resolved upstream at conda-forge.

**Comment by larsoner** [comment]
From one comment, it sounds like the Ubuntu equvialent of `sudo yum install xcb-util-wm xcb-util-keysyms xcb-util-image xcb-util-renderutil` should do it. Based on what my Ubuntu system has, this might do it:
```
sudo apt install libxcb-util1 libxcb-keysyms1 libxcb-image0 libxcb-render-util0
```

**Comment by larsoner** [comment]
> @larsoner Cann we safely assume that our users will have the required xcb libraries installed on their system?

Yes, IIUC if you have GUI/X-based system you'll have these already. It's just these headless/CI machines that have these issues. Users with headless machines shouldn't be using Qt (at least in our setups)...

**Comment by hoechenberger** [comment]
Argh, still not working

**Comment by larsoner** [comment]
You should add the `LD_DEBUG` line, it will tell you what's missing.

You could also copy all the `xcb` libs from circleCI line I linked above. It's likely to make it work

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -303,6 +303,9 @@ jobs:
           mamba env config vars list
           python -c "import os; assert 'PYTHONNOUSERSITE' in os.environ; assert os.environ['PYTHONNOUSERSITE'] == '1'"
 
+          echo "Checking whether (Py)Qt is working"
+          LD_DEBUG=libs python -c "from PyQt5.QtWidgets import QApplication, QWidget; app = QApplication([])"
```
[the remark this answers is not in the document]
Comment (hoechenberger) [reply .github/workflows/build.yml:307]: … I hate everything about this and I'm now going to grab a coffee and grumble about headless CIs, Linux, and conda-forge ruining my day until I get back home! 😤🤬😭☕️

**Comment by hoechenberger** [comment]
Ohhh … that was all? Neat 😍 thanks!

**Comment by larsoner** [comment]
Yeah it should just be in its own step in the pipeline so that its output is collapsed separately from the others like `conda info` and `mamba list`. We'll see if it works now -- it should, but if it doesn't, the end of the `LD_DEBUG=libs` should make it clear which lib it couldn't load/find when trying to use the `xcb` backend

**Comment by larsoner** [comment]
Thanks @hoechenberger !

**Comment by hoechenberger** [comment]
@larsoner We may be able to revert at least parts of this very soon:

https://github.com/conda-forge/qt-main-feedstock/pull/39

**Pull request merged**